k8s/elasticsearch-deployment.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: elasticsearch
+  labels:
+    app: elasticsearch
+spec:
+  selector:
+    matchLabels:
+      app: elasticsearch
+  replicas: 1
```

Comment:
perhaps it would be useful to also give a multi-node elastic setup example, given that the argilla-server can also autoscale.